client/app/scripts/superdesk/editor/editor.js

```diff
@@ -271,28 +389,51 @@ angular.module('superdesk.editor', [])
             templateUrl: 'scripts/superdesk/editor/views/editor.html',
             link: function(scope, elem, attrs, ngModel) {
 
-                var editorElem;
+                var editorElem, spelling = false, timeout;
```

Comment:
This can be confusing for Python developers, it reads as if you were doing this:

```javascript
var editorElem = false;
var spelling = timeout;
```

I suggest having each variable declaration in its own line for better clarity.